File: src/AADDemo.API/Controllers/CarController.cs
using Microsoft.AspNetCore.Authorization;
using Microsoft.AspNetCore.Cors;
using Microsoft.AspNetCore.Mvc;

namespace AADDemo.Controllers
{
  [Authorize]
  [EnableCors("MyPolicy")]
  [ApiController]
  [Route("[controller]")]
  public class CarController : ControllerBase
  {
    [HttpGet] public string Get() => "Honda Legend";
  }
}

File: src/AADDemo.API/Controllers/CarController.fs
namespace AADDemo.API2.Controllers

open Microsoft.AspNetCore.Mvc
open Microsoft.AspNetCore.Authorization
open Microsoft.AspNetCore.Cors

[<Authorize>]
[<EnableCors("AllowAllCors")>]
[<Route("[controller]")>]
type CarController () =
    inherit ControllerBase()

    [<HttpGet>]
    member __.Get() = "Honda Legend" |> fsharpJson